crud.py

```diff
@@ -0,0 +1,75 @@
+from sqlalchemy.orm import Session
+
+import models
+import schemas
+
+
+def get_all_authors(db: Session, limit: int, offset: int):
+    return db.query(models.DBAuthor).limit(limit).offset(offset)
+
+
+def get_all_books(db: Session, limit: int, offset: int):
+    return db.query(models.DBBook).limit(limit).offset(offset)
+
+
+def get_author_by_name(db: Session, name: str):
+    return (
+        db.query(models.DBAuthor).filter(models.DBAuthor.name == name).first()
+    )
+
+
+def get_book_by_title(db: Session, title: str):
+    return (
+        db.query(models.DBBook).filter(models.DBBook.title == title).first()
+    )
+
+
+def get_author(db: Session, author_id: int):
+    return (
+        db.query(models.DBAuthor).
+        filter(models.DBAuthor.id == author_id).first()
+    )
+
+
+def get_book(db: Session, book_id: int):
+    return db.query(models.DBBook).filter(models.DBBook.id == book_id).first()
+
+
+def create_author(db: Session, author: schemas.AuthorCreate):
+    db_author = models.DBAuthor(
+        name=author.name,
+        bio=author.bio,
+    )
```

Comment:
consider this, if you're not doing something special with them:
`**author.model_dump()`